app/models/concerns/annotating/managed_by.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Annotating
+  module ManagedBy
+    extend ActiveSupport::Concern
+
+    def managed_by
+      value_of_annotation("managed")
```

Comment:
I would rather choose the name of annotation - `managed_by`, then it's more clear that a value should be something meaningful, and not `true/false` for example

And there would be less confusion between the name of the annotation and the accessor.

Reply:
Done in https://github.com/3scale/porta/pull/3857/commits/3a7e37575667f08c98954ad7c1a1bd23c3114b6c

Reply:
I hope @akostadinov agrees :sweat_smile:

Reply:
I'm not sure this is an improvement but I'm also not bothered by the change